CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,255 @@
+
+   * Harassment of any participants in any form
+   * Deliberate intimidation, stalking, or following
+   * Logging or taking screenshots of online activity for harassment purposes
+   * Publishing others' private information, such as a physical or electronic address, without explicit permission
+   * Violent threats or language directed against another person
+   * Incitement of violence or harassment towards any individual, including encouraging a person to commit suicide or to engage in self-harm
+   * Creating additional online accounts in order to harass another person or circumvent a ban
+   * Sexual language and imagery in online communities or in any conference venue, including talks
+   * Insults, put downs, or jokes that are based upon stereotypes, that are exclusionary, or that hold others up for ridicule
+   * Excessive swearing
+   * Unwelcome sexual attention or advances
+   * Unwelcome physical contact, including simulated physical contact (eg, textual descriptions like "hug" or "backrub") without consent or after a request to stop
+   * Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others
+   * Sustained disruption of online community discussions, in-person presentations, or other in-person events
+   * Continued one-on-one communication after requests to cease
+   * Other conduct that is inappropriate for a professional audience including people of many different backgrounds
+Community members asked to stop any inappropriate behavior are expected to comply immediately.
+
+
+
+## Manim Community Online Spaces
+
+This Code of Conduct applies to the following online spaces:
+
+- The [ManimCommunity GitHub Organization](https://github.com/ManimCommunity) and all of its repositories
+
+- The Manim [Discord](https://discord.gg/mMRrZQW)
+
+- The Manim [Reddit](https://www.reddit.com/r/manim/)
+
+- The Manim [Twitter](https://twitter.com/manim\_community/)
+
+This Code of Conduct applies to every member in official Manim Community online spaces, including:
+
+- Moderators
+
+- Maintainers
+
+- Developers
+
+- Reviewers
+
+- Contributors
+
+- Users
+
+- All community members
+
+
+
+## Consequences
+
+If a member's behavior violates this code of conduct, the Manim Community Code of Conduct team may take any action they deem appropriate, including, but not limited to: warning the offender, temporary bans, deletion of offending messages, and expulsion from the community and its online spaces. The full list of consequences for inappropriate behavior is listed below in the Enforcement Procedures.
+
+
+
+Thank you for helping make this a welcoming, friendly community for everyone.
+
+
+
+## Contact Information
+
+If you believe someone is violating the code of conduct, or have any other concerns, please contact a Manim Community Dev, or Moderator immediately. They can be reached on Manim's Community [Discord](https://discord.gg/mMRrZQW).
+
+
+
+
+
+## **Enforcement Procedures**
```

Comment:
```suggestion
<hr style="border:2px solid gray"> </hr>
<hr style="border:2px solid gray"> </hr>


## Enforcement Procedures
```

Reply:
Is including the same line twice a typo (i.e. `<hr style="border:2px solid gray"> </hr>`)? If this section is made at a separate level from the Code of Conduct, then I believe all the sections that follow `## Enforcement Procedures` should be at level 3 (e.g. `###`) 

Also, could you rerun the docs build? It failed due to timeout but it looks like this shouldn't be a sphinx error.